spring-cloud-kubernetes-loadbalancer/src/main/java/org/springframework/cloud/kubernetes/loadbalancer/KubernetesLoadBalancerProperties.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.kubernetes.loadbalancer;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author Piotr Minkowski
+ */
+@ConfigurationProperties(prefix = "spring.cloud.kubernetes.loadbalancer")
+public class KubernetesLoadBalancerProperties {
+
+	/**
+	 * Ribbon enabled,default true.
+	 */
+	private Boolean enabled = true;
+
+	/**
+	 * {@link KubernetesLoadBalancerMode} setting ribbon server list with ip of pod or
```

Comment:
The comment refers to Ribbon. Also, the functionality described in the comment is missing and should be added.

Reply:
In my opinion it should be removed. I do not see such a property for load balancer in any other Spring Cloud project like for example Spring Cloud Consul. Maybe it is reasonable to add for example property `spring.cloud.loadbalancer.enabled` and handle it on the level of spring cloud commons? I know that such feature were implemented in `spring-cloud-kubernetes-ribbon`, but it has slightly different approach to load balancing than other Spring cloud projects.